src/saga/mergeUtils.js

```diff
@@ -28,6 +27,9 @@ function checkEmpty(row) {
 }
 
 function handleOriginUndefinedRow(aRow, bRow, sheetName, possibleConflictType, rowIndex) {
+    //TODO: This functionality can be integrated into the original functionality relativly cleanly,
```

Comment:
What is the original functionality? If you can, please clean up this TODO so it can stand alone and be parsed.